doc/rst/language/spec/generics.rst

```diff
@@ -737,9 +741,10 @@ Parameters in Generic Types
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If a class or record defines a parameter field, the class or record is
-generic over the value that is bound to that field. The field can be
-accessed from a class or record instance or from the instantiated class
-or record type itself.
+generic over the value that is bound to that field.  Similarly to
+``type`` fields, a ``param`` field is always generic, whether or not it
```

Comment:
I think you meant "Similarly to ``param`` fields, a ``type`` field is always generic...", since the previous sentence already talks about param fields